docs/installation/observability/metrics.rst

```diff
@@ -0,0 +1,214 @@
+.. _installation_observability_metrics:
+
+=======
+Metrics
+=======
+
+Open Klant produces application metrics (using Open Telemetry).
+
+.. note:: The exact metric names that show up may be transformed, e.g. Prometheus replaces
+   periods with underscores, and processing pipelines may add prefixes or suffixes.
```

Comment:
@danielmursa-dev @sergei-maertens is this transformation a fixed pattern? E.g. if I want to query on `auth.logouts`, in PromQL that would be `otel_auth_logouts_total`?